tests/co19/co19-dart2js.status

```diff
@@ -949,7 +970,26 @@ LibTest/typed_data/Int32x4/operator_OR_A01_t01: RuntimeError # Please triage thi
 LibTest/typed_data/Int8List/sublist_A02_t01: Pass, Slow # Please triage this failure.
 LibTest/typed_data/Uint8ClampedList/map_A02_t01: Pass, Slow # Please triage this failure.
 
+[ $compiler == dart2js && $runtime == chrome && $system != linux]
```

Comment:
Please put the specific runtime == chrome sections after the general runtime == chrome section, and make sure they are merged.